packages/scene-composer/src/components/panels/scene-settings/SceneTagSettingsEditor.tsx

```diff
@@ -37,62 +36,38 @@ export const SceneTagSettingsEditor: React.FC = () => {
           ...getSceneProperty(KnownSceneProperty.ComponentSettings),
           [KnownComponentType.Tag]: newTagSettings,
         };
-
         setSceneProperty(KnownSceneProperty.ComponentSettings, newComponentSettings);
       }
       setTagSettings(newTagSettings);
     },
-    [tagSettings, getSceneProperty, setSceneProperty],
+    [internalScale, tagSettings, getSceneProperty, setSceneProperty],
```

Comment:
`internalScale` is not used in this function

Reply:
nit: I am pretty sure there are lint rules to do this, can we have a task at least to go turn those on so we don't need to rely on Harry to tell us? 🥇

Reply:
Linter implemented in a separate effort (#2779)